src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -401,6 +407,30 @@ private void AddCredentialToCache(Uri serverUri, ulong authAvail, NetworkCredent
             }
         }
 
+        private void AddResponseCookies(Uri serverUri, HttpResponseMessage response)
+        {
+            if (!_useCookie)
```

Comment:
@stephentoub It's possible when you say this you found a different bug:

>I'm unclear as to what the right default should be for HttpClientHandler.UseCookies. It looks like WinHttpHandler maps UseCookies to _cookieUsePolicy, which WinHttpClientHandler defaults to UseInternalCookieStoreOnly, but then the wrapping HttpClientHandler's UseCookies will by default return false, because its getter compares it to UseSpecifiedCookieContainer 

Regarding this code in HttpClientHandler.Windows.cs:

```c#
        public bool UseCookies
        {
            get { return (_winHttpHandler.CookieUsePolicy == CookieUsePolicy.UseSpecifiedCookieContainer); }
            set { _winHttpHandler.CookieUsePolicy = value ? CookieUsePolicy.UseSpecifiedCookieContainer : CookieUsePolicy.IgnoreCookies; }
        }
```